### PR TITLE
Update ts-conda-build to 0.4

### PR DESCRIPTION
**Pull request by couger01**
Fix import path issue in run_athexapod executable and mark as executable.
Update ts-conda-build in test section to 0.4.

### CODE REVIEW

File: doc/news/DM-43481.bugfix.1.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Make run_athexapod executable and fix import path.
```
Comment (wvreeven) [inline doc/news/DM-43481.bugfix.1.rst]: It looks like this change has not been pushed to GitHub. Can you check please?

**Review by wvreeven — CHANGES_REQUESTED** [review]
It looks like some code changes have not been pushed to GitHub. See my comment.

File: doc/news/DM-43481.bugfix.1.rst (hunk 1 above)
Comment (couger01) [reply doc/news/DM-43481.bugfix.1.rst]: Actually I just realized that the change was for the athexapod and not the atwhitelight please disregard and have removed the news fragement.

**Review by wvreeven — APPROVED** [review]
LGTM

**Pull request merged**